modules/installation-creating-azure-control-plane.adoc

```diff
@@ -54,3 +65,30 @@ $ az deployment group create -g ${RESOURCE_GROUP} \
 <2> The SSH RSA public key file as a string.
 <3> The name of the private DNS zone to which the control plane nodes are attached.
 <4> The base name to be used in resource names; this is usually the cluster's infrastructure ID.
+endif::azure[]
+ifdef::ash[]
+[source,terminal]
+----
+$ az deployment group create -g ${RESOURCE_GROUP} \
+  --template-file "<installation_directory>/05_masters.json" \
+  --parameters masterIgnition="${MASTER_IGNITION}" \ <1>
+  --parameters sshKeyData="${SSH_KEY}" \ <2>
+  --parameters baseName="${INFRA_ID}"<3>
+  --parameters masterVMSize="Standard_D4_v2" \
+  --parameters diskSizeGB="1023" \
```

Comment:
the vm and disk size can be omitted for simplicity because these are the default values